src/Extensions/index.ts

```diff
@@ -0,0 +1,3 @@
+import.meta.glob(['./**/*.ts', './**/*.tsx'], { eager: true })
```

Comment:
If I'm not mistaken, the consequence of this, is that _all_ `contentDump` is loaded on every editor visit of the page. `contentDump` should only be loaded, once the restore process is started.

Reply:
The recursive imports only create maps of filename to promise that load the content dump items on demand (after the dialog opens).
However, I'll make the top-level import more specific.